docs/source/tensors.rst

```diff
@@ -45,8 +48,35 @@ quantized 4-bit integer (unsigned) [3]_ ``torch.quint4x2``
   number of exponent bits as ``float32``
 .. [3]
   quantized 4-bit integer is stored as a 8-bit signed integer. Currently it's only supported in EmbeddingBag operator.
-
-:class:`torch.Tensor` is an alias for the default tensor type (:class:`torch.FloatTensor`).
+.. [4]
+  Unsigned types asides from ``uint8`` are currently planned to only have
+  limited support in eager mode (they primarily exist to assist usage with
```

Comment:
Do we want to specify "eager mode" here? Do we actually plan to generally support these in compile?

Reply:
Yeah, I intend for these to be well supported in compile

Reply:
can this talk about op support instead of eager vs compile?  I should be able to use tensor subclass types in eager mode, right?